### PR TITLE
Update language.c

### PR DESCRIPTION
**Pull request by AntonioDePau**
Added Spanish and French translations for "Back" option.
Also, if someone can take a look at the comment I added for the "Open" option and let me know if "Open" is used as a adjective here ("This folder is open") or as a verb ("Open this folder?").

If you need anything to be translated to French and/or Spanish, feel free to contact me!

(Also, sorry if this isn't the way to suggest changes, or if I did it wrong, please tell me how to it better, for next time!)

### CODE REVIEW

File: common/language.c [hunk 1]
```diff
@@ -264,7 +264,7 @@ const char* const g_strings[StrId_Max][16] =
         STR_EN("Open"),
         STR_ES("Abierto"),
         STR_DE("Öffnen"),
-        STR_FR("Ouvrir"),
+        STR_FR("Ouvrir"),   //if "Open" is used as an adjective here (just like Spanish), and not a verb, consider changing "Ouvrir" to "Ouvert", because "Ouvrir" is the verb "To open"
```
Comment (fincs) [inline common/language.c]: This is a verb. Spanish translation should have been `Abrir`...

Comment (AntonioDePau) [reply common/language.c]: Thanks, then STR_IT and STR_PT are probably wrong too. I'm not fluent/confident enough to make the changes for those two. But I'll do STR_ES then.

**Comment by yellows8** [comment]
Can the remaining "Back" language entries be added with this PR?(Like, others could comment with text for {language})

**Comment by AntonioDePau** [comment]
> Can the remaining "Back" language entries be added with this PR?(Like, others could comment with text for {language})

Done. Can someone please double check, someday, the languages mentioned in the comment?
I am only 100% certain for FR and ES, and 90% for IT and PR.

**Comment by Ryuzaki-MrL** [comment]
PT is correct now.

**Comment by FabioArdis** [comment]
The Italian translation of "back" is "Indietro".

**Comment by baeby** [comment]
Help me

**Comment by thisismike97** [comment]
The STR_NL one should just be "Terug" instead of "teruggaan"

**Pull request merged**